tests/Agent/IntegrationTests/SharedApplications/Common/MultiFunctionApplicationHelpers/NetStandardLibraries/RestSharp/RestSharpExerciser.cs

```diff
@@ -0,0 +1,223 @@
+﻿// Copyright 2020 New Relic, Inc. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+
+using System;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using RestSharp;
+using NewRelic.Agent.IntegrationTests.Shared.ReflectionHelpers;
+using NewRelic.Api.Agent;
+using System.Runtime.CompilerServices;
+
+namespace MultiFunctionApplicationHelpers.NetStandardLibraries.RestSharp
+{
+    [Library]
+    public class RestSharpExerciser
+    {
+        private class Bird
+        {
+            public string CommonName { get; set; }
+            public string BandingCode { get; set; }
+        }
+
+        [LibraryMethod]
+        [Transaction]
+        [MethodImpl(MethodImplOptions.NoOptimization | MethodImplOptions.NoInlining)]
+        public string SyncClient(int port, string method, bool generic)
+        {
+            var myHost = "localhost"; // Request.RequestUri.Host;
+            var myPort = port;
+            var client = new RestClient($"http://{myHost}:{myPort}");
+
+            var endpoint = "api/RestAPI/";
+            var id = 1;
+
+            var requests = GetRequests(endpoint, id);
+            if (generic)
+            {
+                var response = client.Execute<Bird>(requests[method]);
+                if (method == "GET")
+                {
+                    var bird = response.Data;
+                    System.IO.File.AppendAllText(@"C:\IntegrationTestWorkingDirectory\RestAPIController.log", $"SyncClient method={method}, generic={generic} got Bird {bird.CommonName} ({bird.BandingCode})" + Environment.NewLine);
```

Comment:
I'm guessing this was some debugging code?

Reply:
No, at least not intentionally.  This class was basically a duplicate of the existing [RestSharpController](https://github.com/newrelic/newrelic-dotnet-agent/blob/main/tests/Agent/IntegrationTests/Applications/BasicMvcApplication/Controllers/RestSharpController.cs) used by the old test fixture.  I was focusing on keeping it as close to the original as possible to minimize any test hiccups (I still got waylaid by several) and that included leaving in some of the odd output bits like you called out.

Reply:
That is odd. Hard coded paths make me nervous, even if it's just an integration test, but I guess it's no worse than it was.